api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.6.x.xml

```diff
@@ -116,4 +116,9 @@
 		<addForeignKeyConstraint constraintName="medication_dispense_voided_by_fk" baseTableName="medication_dispense" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id"/>
 	</changeSet>
 	
+	<!-- Adding not null constraint to encounter_diagnosis.dx_rank. TRUNK-6081-->
```

Comment:
@wluyima here is the comment I added.

Reply:
That's not what I mean, I mean a liquibase comment, not an xml comment, sorry if I wasn't clear

Reply:
@wluyima I added the necessary liquibase comment.